### PR TITLE
add gpg sign

### PR DESCRIPTION
**Pull request by bitsf**
Change-Id: I58369eb059243e0a059c8be86cd980c2cb42f0bc
Signed-off-by: Ziming Zhang <zziming@vmware.com>

### CODE REVIEW

File: tests/integration.sh [hunk 1]
```diff
@@ -151,6 +151,8 @@ if [[ $DRONE_BRANCH == "master" || $DRONE_BRANCH == *"refs/tags"* || $DRONE_BRAN
     if [[ $DRONE_BUILD_EVENT == "push" ]]; then
         package_offline_installer
         upload_latest_build=true
+        echo -en "$HARBOR_SIGN_KEY" | gpg --import
+        gpg -v -ab -u C0B4115C $harbor_build_bundle
```
Comment (reasonerjt) [inline tests/integration.sh]: make it a var

File: tests/integration.sh [hunk 2]
```diff
@@ -169,8 +171,11 @@ fi
 set -e
 if [ $upload_build == true ]; then
     cp $harbor_build_bundle harbor-offline-installer-latest.tgz
+    cp $harbor_build_bundle.asc harbor-offline-installer-latest.tgz.asc
```
Comment (reasonerjt) [inline tests/integration.sh]: add `{}`

**Comment by xaleeks** [comment]
@stuclem  fyi, all future releases will follow this format of including the tgz.asc key

**Comment by reasonerjt** [comment]
@xaleeks this is build signing, not image signing.

**Pull request merged**